back-end/src/controllers/users.js

```diff
@@ -6,8 +6,16 @@ async function login( req, res, next ) {
   return res.json( await userService.validateLogin( email, password ) )
 }
 
+async function addUser( req, res, next ){
+  const { name, password, email, photo, level, isActive } = req.body;
+
+  return res.json( await userService.addNewUser( name, password, email, photo, level, isActive ) );
```

Comment:
Uma sugestão aqui, é reduzir o número de parâmetros no método addNewUser, como no exemplo:
```suggestion
  const { name, password, email, photo, level, isActive } = req.body;
  const user = {
    name,
    password, 
    email, 
    photo, 
    level, 
    isActive
  }

  return res.json( await userService.addNewUser( user ) );
```

Reply:
Sugestão aceita